recipes/viennarna/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.4.7" %}
-{% set sha256 = "3dbb6c6b54a372193d8106ee5f59442a0d7535838812a0aade09e632e599e4ee" %}
+{% set version = "2.4.8" %}
+{% set sha256 = "1f8a08b1c8e57e4c6610e0e48c66ae692caba4183867cde8eb58ec16c40f78fa" %}
 
 package:
   name: viennarna
   version: {{ version }}
 
 build:
-  number: 5
+  number: 0
   string: "py{{CONDA_PY}}_{{PKG_BUILDNUM}}"
```

Comment:
You can remove that `build/string` line.

Reply:
@s-will do yo need the package string extension for something?

Reply:
Hi @mbargull, thanks for the review. The build string lets us build different packages for the different python versions, which we need to build python bindings for the respective version. Removing the build string thus breaks this functionality. Possibly this is quite a hack as well---would you suggest an alternative mechanism?

Reply:
`conda-build` will automatically take care of that. Either it creates build string with exactly that schema, e.g., `py36_0`, or something like `py36_h789abcd_0` when `python` is in `requirements/host` as well as `requirements/run`. With some older versions of `conda-build` there were few cases where the build strings had to be manually specified, but that shouldn't be needed for the current versions.